acneditor.cpp

```diff
@@ -54,6 +55,8 @@ AcnEditorFactory::AcnEditorFactory()
     setDocumentCreator([]() { return new AcnDocument; });
     setEditorWidgetCreator([]() { return new AcnEditorWidget; });
     setEditorCreator([]() { return new AcnEditor; });
+
+    setCompletionAssistProvider(new AcnCompletionAssistProvider);
```

Comment:
if we introduce new subdir (which is a good idea) maybe we should create some sub-namespace for items in that dir? just for consistency. Then some items in that subdir could have shorter names :)